core-services/release-controller/deploy-ocp-controller-ppc64le.yaml

```diff
@@ -161,3 +161,9 @@ items:
           - --prow-job-kubeconfig=/etc/prowjob-kubeconfig/kubeconfig
```

Comment:
@AlexNPavel what about `deploy-origin-controller.yaml`, that one does not get the flags?

Reply:
Only OCP builds are valid for verifying bugs, not origin, so I only added the flag to OCP for now.

Reply:
What we move to verified would be handled via the configs though, so I guess it wouldn't be a problem.